test/e2e/tests/editor/events.spec.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require('@wordpress/e2e-test-utils-playwright');
+
+test.describe('Events in the Editor', () => {
+	test.beforeEach(async ({ admin, page }) => {
+		await admin.createNewPost({ postType: 'gatherpress_event' });
+		await page.getByLabel('Add title').fill('Change title to allow saving');
+		await page
+			.getByLabel('Block: Event Date')
+			.locator('div')
+			.first()
+			.isVisible();
+		await page.getByRole('heading', { name: 'Date & time' }).isVisible();
+
+		// Open the Document -> Event settings panel.
+		const panelToggle = page.getByRole('button', {
+			name: 'Event settings',
+		});
+
+		if ((await panelToggle.getAttribute('aria-expanded')) === 'false') {
+			// Check if the inserter button is visible
+			await expect(panelToggle).toBeVisible();
+
+			// Ensure the panelToggle is enabled before interacting with it
+			await expect(panelToggle).toBeEnabled();
+
+			// Click the panelToggle
+			// Just click() without forcing led to timeouts, timeouts, timeouts...
+			await panelToggle.click({ force: true });
+		}
```

Comment:
This block is equivalent to:

```suggestion
		// Open the Document -> Event settings panel.
		const panelToggle = page.getByRole('button', {
			name: 'Event settings',
			expanded: false,
		});

		if (await panelToggle.isVisible()) {
			await panelToggle.click();
		}
```

Reply:
Cool, thats a lot more readable.
Thank you!